docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.7'
+
+services:
+  rabbitmq:
+    container_name: some-rabbit
+    ports:
+      - '15672:15672'
+    image: rabbitmq:3-management
+    networks:
+      - my-net
+
+  chocodoc:
+    container_name: choco_doc
+    ports:
+      - '3000:3000'
+      - '7000:7000'
```

Comment:
since we don't start an nREPL, this doesn't do anything. Should we try to start an nREPL? Might be interesting give the purpose for chocolate.